scripts/fetch_ci_binaries.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+"""Fetch executable binary artifacts from the latest Circle CI workflow on main"""
+
+import os
+import sys
+from typing import List, Optional, Tuple
+import requests
+
+HTTP_TIMEOUT = 5
+
+DEVNET_CI_URL = (
+    "https://circleci.com/api/v1.1/project/github/0xSpaceShard/starknet-devnet-rs"
+)
+
+BINARY_BUILD_JOB_PREFIX = "binary-build-"
+
+
+def get_artifact_info(job: dict) -> Optional[Tuple[str, str]]:
+    """
+    Returns the artifact (URL, name) corresponding to job object.
+    Returns None if job not a binary build.
```

Comment:
Returns None if job **is** not a binary build

Reply:
I won't change this, it's programmer English